### PR TITLE
Upgrade PHPUnit and bump to PHP >=7.2

### PR DESCRIPTION
**Pull request by williamdes**
### Description:

- Upgrade PHPUnit
- bump to PHP >=7.2
- removed chmod executable on some files that should not have it (https://github.com/matomo-org/component-decompress/commit/a7413d7872ba7b6af05def5b127d6a7d7024c7cc)

I can re-build the CI on GitHub if this PR is accepted

### CODE REVIEW

**Comment by sgiehl** [comment]
@williamdes thanks for creating this PR. Would you mind to update the travis.yml file to only run tests for PHP 7.2+. Once the tests are passing again I'm happy to approve and merge this.

**Comment by williamdes** [comment]
> @williamdes thanks for creating this PR. Would you mind to update the travis.yml file to only run tests for PHP 7.2+. Once the tests are passing again I'm happy to approve and merge this.

Sure, do you prefer that I migrate them to GH in this PR directly ?

File: tests/PclZipTest.php [hunk 1]
```diff
@@ -34,11 +34,11 @@ public function testRelativePathAttack()
         $unzip = new PclZip($filename);
         $res = $unzip->extract($this->tempDirectory);
         $this->assertEquals(0, $res);
-        $this->assertFileNotExists($this->tempDirectory . $test . '.txt');
-        $this->assertFileNotExists($this->tempDirectory . '../' . $test . '.txt');
-        $this->assertFileNotExists(__DIR__ . '/' . $test . '.txt');
-        $this->assertFileNotExists(__DIR__ . '/../' . $test . '.txt');
-        $this->assertFileNotExists(__DIR__ . '/../../' . $test . '.txt');
+        $this->assertFileDoesNotExist($this->tempDirectory . $test . '.txt');
```
Comment (williamdes) [inline tests/PclZipTest.php:37]: Needs a fix for PHP 7.2 PHPUnit

**Comment by williamdes** [comment]
@sgiehl Fixed 🎉

**Comment by sgiehl** [comment]
Thanks for the contribution @williamdes. Feel free to create a PR to also move the travis tests to a github action.

**Pull request merged**